src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -18,6 +18,52 @@
       ]
     }
   },
+  {
+    "name": "AppService_FunctionApp_HTTPS",
```

Comment:
Remove underscore (or add to others) before FunctionApp to keep the name consistent.